ui/app/services/keyboard.js

```diff
@@ -411,6 +411,7 @@ export default class KeyboardService extends Service {
             command.label === 'Show Keyboard Shortcuts' ||
             command.label === 'Hide Keyboard Shortcuts'
           ) {
+            event.preventDefault();
```

Comment:
Added this so that the letter `n` (from having typed `run`) doesn't show up in the job template editor box (keyboard service eagerly performs a task on keydown, but typing waits for keyup)